src/_includes/macros/util/format/datetime.html

```diff
@@ -0,0 +1,77 @@
+{# ==========================================================================
+
+   format_time()
+
+   ==========================================================================
+
+   Description:
+
+   Format a time for displaying in the format of:
+
+    HH:MM AM ZZZ
```

Comment:
Should be four spaces, if following format in `contact.html` and `url.html`. This is a carry-over format from the old codebase for these macros.